src/api/consumer/lib/import.js

```diff
@@ -65,6 +65,7 @@ export default function importAction(
                                                                           //
           return Promise.resolve(({ dependencies, envDependencies }));    // here we should return { dependencies, envDependencies: [] }
         })
+        .then(allDependencies => consumer.runHook('onImport', allDependencies))
```

Comment:
How can we make sure the hook returns the { dependencies, envDependencies } object for the warning procedure? Do we have to return the same object from all of the hooks? I think we don't want to rely on that, we can use tap (http://ramdajs.com/docs/#tap) to verify the return value, as the hook just makes a side effect.